css/selectors/link.json

```
@@ -70,8 +70,8 @@
               },
               "opera_android": "mirror",
               "safari": {
-                "version_added": false,
-                "notes": "Safari currently matches <code>&lt;link&gt;</code> elements with link pseudo-classes. See <a href='https://webkit.org/b/220740'>Bug: 220740</a>."
+                "version_added": "15",
+                "notes": "Safari used to match <code>&lt;link&gt;</code> elements with link pseudo-classes. See <a href='https://webkit.org/b/227847'>Bug: 227847</a>."
```

Let's remove the note entirely; it's just a repeat of the feature description.

```suggestion
                "version_added": "15"
```